src/lib.rs

```diff
@@ -152,6 +152,29 @@
 //! LD_LIBRARY_PATH=/path/to/mylib/target/debug`. Now, you should get the
 //! expected output `Hello, josh!` from your Java class.
 //!
+//! If you build with the `invocation` feature, there is yet another dependency:
```

Comment:
I'd rewrite this paragraph:
1. As a separate section.
1. As user-oriented (but the relevant impl details can be moved to the build-script).

E.g. (needs refs and polishing)
```
### Launching JVM from Rust

If you need to use the part of the Invocation API 
(link to the docs) that allows to launch a JVM 
from a native process, you must enable "invocation" feature.
The application will require linking to the dynamic `jvm` 
library, which is distributed with the JVM.

During build time, the JVM installation is determined:
1. By `JAVA_HOME` environment variable, if it is set.
2. Otherwise — from `java` output (*please move the specifics to the build-file documentation*).
It is recommended to set `JAVA_HOME` to have reproducible builds, especially, in case of multiple VMs installed.

At application run time, you must specify the path 
to the `jvm` library so that the loader can locate it.
* On Windows, append the path to `jvm.dll` to `PATH` environment variable.
* On MacOS, append the path to `libjvm.dylib` to `LD_LIBRARY_PATH` environment variable.
* On Linux, append the path to `libjvm.so` to `LD_LIBRARY_PATH` environment variable.

The exact relative path to `jvm` library is version-specific.
```